apps/backend/src/parse/parse.service.ts

```diff
@@ -336,39 +358,90 @@ export class ParseService {
         if (requiredFile !== undefined) {
           return requiredFile.fileType;
         }
+        // TODO email? Do we include the message in this case?
         throw new Error('Unknown file type: ' + filename);
       })();
+      return { currentRule, fileType, programRules };
+    } catch (e) {
+      this.appLogger.error(e);
+      // TODO email? Do we include the message in this case?
```

Comment:
I think this is pretty handily covered by the above errors? I think no email then.